recipes/strawberryperl/all/conanfile.py

```diff
@@ -40,4 +40,5 @@ def package_info(self):
         self.output.info("Appending PATH environment variable: %s" % bin_path)
         self.env_info.PATH.append(bin_path)
 
+        self.user_info.perl = os.path.join(self.package_folder, "bin", "perl.exe").replace("\\", "/")
         self.deps_user_info.perl = os.path.join(self.package_folder, "bin", "perl.exe").replace("\\", "/")
```

Comment:
I don't have any idea, if this line does anything, as OpenSSL uses rootpath. I kept it just in case.

Reply:
I would say the usage of `deps_user_info` here is not documented in Conan. I would vote to remove the line... but maybe it is worth to check consumer recipes before. 
cc/ @SSE4

Reply:
`git blame` points at me.
I think the `deps_user_info` was a typo and can be removed..

Reply:
So, I guess the line can get removed. This should then also fix #4211 and https://github.com/conan-io/conan/issues/8313

Reply:
please link the issue!

Reply:
Done